src/java/org/apache/cassandra/index/sai/disk/io/IndexOutputWriter.java

```diff
@@ -37,6 +37,9 @@ public class IndexOutputWriter extends IndexOutput
 {
     private static final Logger logger = LoggerFactory.getLogger(IndexOutputWriter.class);
 
+    // the byte order of `out`'s native writeX operations doesn't matter,
+    // because we only use `write(byte[])` and `writeByte` methods. IndexOutput calls these
+    // (via DataOutput) with methods that enforce LittleEndian-ness.
```

Comment:
Similar more navigable form of this:

```
/**
 * The byte order of {@link SequentialWriter}'s native write operations doesn't matter,
 * because we only use {@link SequentialWriter#write(byte[])} and {@link SequentialWriter#writeByte} methods.
 * {@link IndexOutput} calls these (via {@link DataOutput}) with methods that enforce LittleEndian-ness.
 */
```